addons/stock_account/models/stock_move.py

```diff
@@ -256,6 +256,9 @@ def _action_done(self, cancel_backorder=False):
 
         res = super(StockMove, self)._action_done(cancel_backorder=cancel_backorder)
 
+        # '_action_done' might have deleted some exploded stock moves
+        valued_moves = {value_type: moves.exists() for value_type, moves in valued_moves.items()}
```

Comment:
it would be better to return all the moves from action_done or confirm before the super because you will miss the new kit's moves no?